site/src/components/card.md

```diff
@@ -0,0 +1,203 @@
+- `.card-header` - an optional section on top of the card.
+- `.card-template` - a highly prescriptive layout for the card's main contents. Not intented for compatibilty with `.card-content`.
+- `.card-content` - a free-form box within which a card's main content can be placed. Intended for use instead of `.card-template` not in addition to it.
+- `.card-footer` - an optional section at the bottom of the card. Usually contains actions relevant to the card's subject.
+
+Each of these subcomponents has their own optional subcomponents. They are discussed below.
+
+## The main content area of card
+
+In styling the main content area of a card, it's recommended you look first to `.card-template`, and if that fails, fall back to the less prescriptive `.card-content`.
+
+Both `.card-template` and `.card-content` support the use of several subcomponents. Because they can be used in both `.card-template` and `.card-content`, they are no namespaces by their parent elements, but rather only by `.card-`
+
+- `.card-supertitle` - a uppercase piece of text above the title.
+- `.card-title` - the title of the card.
+
+The following can be used only inside `.card-content`.
+
+- `.card-content-description` - a body of text truncated at four lines.
+
+The following can be used only inside `.card-template`.
+
+- `.card-template-icon` - a grid area that houses a small image.
+- `.card-template-detail` - a catch all that sits next to the card icon.
+
+```html
+<article class="card width-300">
+	<div class="card-template">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+		<img
+			aria-hidden="true"
+			class="card-template-icon"
+			src="https://docs.microsoft.com/en-us/learn/achievements/review-microsoft-azure-pricing-slas-lifecycles.svg"
+		/>
+		<div class="card-template-detail">
+			<p>Additonal details might be listed here.</p>
+		</div>
+		</div>
+	</div>
+</article>
+```
+
+A similar example using `.card-content`, including its only optional subcomponent `card-content-description`.
+
+```html
+<article class="card width-300">
+	<div class="card-content">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+		<p class="card-content-description">Additonal details might be listed here.</p>
+	</div>
+</article>
+```
+
+## Card footer
+
+The `.card-footer` class is an optional. It is often used to display actions related to the card's subject (like "save" or "follow"). Or for display information that doesn't fit well elsewhere, such as progress. It has a child component called `.card-footer-item`. Place `.card-footer-left` or `.card-footer-right` on `.card-footer` (or use flex atomics) to align its inner items as you see fit. Note that the `.buttons` class is well supported in this context and can be added if space between buttons is desired. It's recommended that you use small buttons (`.button-sm`) if placing buttons in this area.
```

Comment:
"The `.card-footer` class is an optional" - sounds like it's missing something at the end.

Reply:
"Or for displaying information"

Reply:
Going to merge with the next sentence